user_category.py

```diff
@@ -7,21 +7,30 @@
 #pick current year
 current_year = current_time.year
 
-#prompt user to enter their year of birth
-birth_year = int(input("Enter your year of birth (eg 2000):"))
-
 #make sure user enters only int values
-def only_int(birth_year):
-    try: 
-        if type(birth_year) == "int": 
-            #current the determinant value
-            user_category_value = current_year - birth_year
-            return user_category_value
-        else: 
-           print("Sorry, please enter year in numbers")
-           return False
-    except ValueError:
-        return False
+def number():
+    while True:
+        try:
+            #prompt user to enter their year of birth
+            birth_year = int(input("Enter your year of birth (eg 2000):"))
+        except ValueError:
+            print("Sorry, please enter year in numbers")
+            continue
+        if birth_year < 0:
+            print("Your Entered a negative value")
+            continue
+        elif birth_year > current_year:
+            print("You cannot be born in future ")
+            continue
+        elif birth_year < 1800:
+            print("You must be dead by now")
+            continue
+        else:
+            return birth_year
+            break
```

Comment:
This break is redundant. It will never be reached. Once a function finds a return statement. It doesn't execute anything below it